.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,18 @@
+Thank you so much for your PR! To help us review, fill out the form to the best of your ability. Also, please make use of the development guide at http://matplotlib.org/devdocs/devel/index.html
+
+<!--- Provide a general summary of your changes in the Title above, for example "Raises ValueError on Non-Numeric Input to set_xlim". Please avoid non-descriptive titles such as "Addresses issue #8576"-->
+
+## PR Summary
+<!--- Please provide at least 1-2 sentences describing the pull request in detail. Why is this change required? What problem does it solve? -->
+<!--- If it fixes an open issue, please link to the issue here. -->
+
+## PR Checklist
+- [ ] Code is tested using Py.test on Py2 and Py3
```

Comment:
it's pytest (no dot) these days

it's may be a bit harsh to request testing on both py2 and py3 (plus you also need to check somewhat antique versions of numpy) so I'd just say "unit tests are present" (travis will catch failures anyways)  (for example I typically only run tests on py3.6+latest numpy and just let travis tell me if anything else needs to be corrected)